decidim-initiatives/lib/decidim/initiatives/engine.rb

```diff
@@ -19,7 +19,16 @@ class Engine < ::Rails::Engine
         get "/initiative_type_scopes/search", to: "initiatives_type_scopes#search", as: :initiative_type_scopes_search
         get "/initiative_type_signature_types/search", to: "initiatives_type_signature_types#search", as: :initiative_type_signature_types_search
 
-        resources :create_initiative
+        resources :create_initiative do
+          collection do
+            match :select_initiative_type, via: [:get, :put]
+            match :previous_form, via: [:get, :put]
+            match :show_similar_initiatives, via: [:get, :put]
+            match :fill_data, via: [:get, :put]
+            match :promotal_committee, via: [:get, :put]
+            match :finish, via: [:get, :put]
```

Comment:
(This is unrelated to my discussion on the HTTP methods separation):

Some of these action don't have any forms in their views, so we don't need to use PUT here: 
```suggestion
            match :select_initiative_type, via: [:get, :put]
            match :previous_form, via: [:get, :put]
            get :show_similar_initiatives
            match :fill_data, via: [:get, :put]
            get :promotal_committee
            get :finish
```